sklearn/metrics/ranking.py

```diff
@@ -217,6 +217,80 @@ def _binary_uninterpolated_average_precision(
                                  sample_weight=sample_weight)
 
 
+def detection_error_tradeoff_curve(y_true, y_score, pos_label=None,
+                                   sample_weight=None):
+    """Compute error rates for different probability thresholds.
+
+    Note: This implementation is restricted to the binary classification task.
```

Comment:
Is it commonly applied to non-binary tasks?

Reply:
Not that I am aware of, but I am anything but an expert on that domain. By their nature of comparing error types they are best suited for evaluation of binary detection tasks.

Reply:
Maybe you can reword so this does not seem like a fault of the implementation